group_vars/openstack.yml

```diff
@@ -19,6 +19,4 @@ cluster_ssh_user: rocky
 
 # Set the size of the state volume to metrics_db_maximum_size + 10
 state_volume_size: "{{ metrics_db_maximum_size + 10 }}"
-
-state_volume_device_path: "{{ cluster_state_volume_device_path | default('/dev/vdb') }}"
```

Comment:
Renaming this variable and changing its format will break existing deployments that have it set. It might not be a problem for now, but something to be wary of for future changes.